tools/autograd/derivatives.yaml

```diff
@@ -1486,3 +1516,47 @@
 # PackedSequence helpers
 - name: _pack_padded_sequence(Tensor input, Tensor lengths, bool batch_first)
   input: _pack_padded_sequence_backward(grad, input.sizes(), result1, batch_first)
+
+# TH wrappers
+
+- name: eq(Tensor self, Scalar other)
+  output_differentiability: [False]
+
+- name: eq(Tensor self, Tensor other)
+  output_differentiability: [False]
+
+- name: ge(Tensor self, Scalar other)
+  output_differentiability: [False]
+
+- name: ge(Tensor self, Tensor other)
+  output_differentiability: [False]
+
+- name: gt(Tensor self, Scalar other)
+  output_differentiability: [False]
+
+- name: gt(Tensor self, Tensor other)
+  output_differentiability: [False]
+
+- name: le(Tensor self, Scalar other)
+  output_differentiability: [False]
+
+- name: le(Tensor self, Tensor other)
+  output_differentiability: [False]
+
+- name: lt(Tensor self, Scalar other)
+  output_differentiability: [False]
+
+- name: lt(Tensor self, Tensor other)
+  output_differentiability: [False]
+
+- name: ne(Tensor self, Scalar other)
+  output_differentiability: [False]
+
+- name: ne(Tensor self, Tensor other)
+  output_differentiability: [False]
+
+- name: multinomial(Tensor self, int64_t num_samples, bool replacement, *, Generator generator)
+  output_differentiability: [False]
+
+- name: is_nonzero(Tensor self)
```

Comment:
I don't really understand why you need to make the changes to this file.

For example, is_nonzero returns a bool -- how can that possibly be differentiable?

Reply:
I don't think it's required for is_nonzero, i'll remove it. But the others are required because they were initially marked not differentiable because they are BoolTensors or IndexTensors in Declarations.cwrap. Now that we don't have that information, we need to put it here.